go/kbfs/libkbfs/disk_block_cache.go

```diff
@@ -196,6 +197,7 @@ func newDiskBlockCacheLocalFromStorage(
 	}()
 	blockDbOptions := *leveldbOptions
 	blockDbOptions.CompactionTableSize = defaultBlockCacheTableSize
+	blockDbOptions.BlockSize = defaultBlockCacheBlockSize
```

Comment:
Why wasn't there a default before?

Reply:
There is an existing default, but it's shared amoung all leveldb instances and is a little small for the block db: https://github.com/keybase/client/blob/8e488dd1b95d7626d7ae8d3fc88c56e9f3e054f8/go/kbfs/libkbfs/leveldb.go#L31